LibertyDSNP/schemas
# Update for Frequency 1.4.0 and DSNP 1.2.0

Problem
=======

Needed to add the schemas for the upcoming DSNP 1.2.0 release.

Change summary:
---------------
* Updated the DSNP schemas for 1.2.0
* Updated to Frequency 1.5.1
* Cleaned up the deploy and read logic and made it easier to add new DSNP Schemas

Steps to Verify:
----------------
1. Run Frequency 1.4.0+ locally
1. `npm run deploy`
1. `npm run read`
1. See that they all deployed and are working!
